modules/canvaskit/compile.sh

```diff
@@ -36,12 +36,15 @@ if [[ $@ != *force_tracing* ]] ; then
   EXTRA_CFLAGS+="\"-DSK_DISABLE_TRACING\","
 fi
 
+IS_DEBUG="false"
 if [[ $@ == *debug* ]]; then
   echo "Building a Debug build"
   EXTRA_CFLAGS="\"-DSK_DEBUG\","
   RELEASE_CONF="-O0 --js-opts 0 -s DEMANGLE_SUPPORT=1 -s ASSERTIONS=1 -s GL_ASSERTIONS=1 -g3 \
                 --source-map-base /node_modules/canvaskit/bin/ -DSK_DEBUG --pre-js $BASE_DIR/debug.js"
   BUILD_DIR=${BUILD_DIR:="out/canvaskit_wasm_debug"}
+  IS_DEBUG="true"
+  IS_OFFICIAL_BUILD="false"
```

Comment:
What does IS_OFFICIAL_BUILD do?

Reply:
It is kind of isRelease flag.
So to make it debug, it needs to be set to false...

is_official_build=${IS_OFFICIAL_BUILD} \

It controls things like:

if (!is_official_build) {
  skia_target_default_configs += [ "//gn/skia:warnings" ]
}

skia_header_target_default_configs = []
if (!is_official_build) {
  skia_header_target_default_configs +=
      [ "//gn/skia:warnings_for_public_headers" ]
}